app/helpers/validation_helper.rb

```diff
@@ -233,15 +235,17 @@ def validate_between_dates(record, attribute, earliest_date, latest_date)
   # Format validation errors from the model to be more human-readable
   def format_model_validation_errors(resource)
     resource.errors&.messages&.each_with_object([]) do |(attribute, errors), messages|
-      next unless VALIDATION.key?(attribute)
+      next unless VALIDATION.key?(attribute) || attribute == :base
 
       # NOTE: If the value is a date, the typecast value may not correspond to original user input, so get value_before_type_cast
-      value = VALIDATION[attribute][:checks].include?(:date) ? resource.public_send("#{attribute}_before_type_cast") : resource[attribute]
-      msg_header = (value ? " Value '#{value}' for " : '') + "'#{VALIDATION[attribute][:label]}'"
+      unless attribute == :base
+        value = VALIDATION[attribute][:checks].include?(:date) ? resource.public_send("#{attribute}_before_type_cast") : resource[attribute]
+        msg_header = (value ? "Value '#{value}' for " : '') + "'#{VALIDATION[attribute][:label]}' "
+      end
       errors.each do |error_message|
         # Exclude the actual value in logging to avoid PII/PHI
         Rails.logger.info "Validation Error on: #{attribute}"
-        messages << "#{msg_header} #{error_message}"
+        messages << "#{msg_header}#{error_message}"
```

Comment:
```suggestion
        messages << "#{msg_header} #{error_message}".trim
```
And then also can remove the trailing space from `msg_header`.

Reply:
Good call, addressed in https://github.com/SaraAlert/SaraAlert/pull/739/commits/5efb740a25cca0ddf4ffd00128610e3a3dd68f6b.

Reply:
Lol not sure why my brain put `trim` there, but you got it... thanks